docs/content/preview/yugabyte-voyager/release-notes.md

```diff
@@ -13,6 +13,34 @@ type: docs
 
 What follows are the release notes for the YugabyteDB Voyager v1 release series. Content will be added as new notable features and changes are available in the patch releases of the YugabyteDB v1 series.
 
+## Versioning
+
+Starting with this release, Voyager uses a new versioning format: `YYYY.M.N` (for example, `2025.5.2`), where:
+
+- `YYYY` represents the release year
+- `M` represents the release month
+- `N` represents the number of the release in that month
+
+The previous version used the old format (`v1.8.17`). This update aligns versioning with Yugabyte Voyager release cadence and makes it easier to track updates over time.
+
+## v2025.5.2 - May 20, 2025
+
+### Enhancements
+
+- Voyager now automatically runs `assess-migration` when `export schema` is executed before it.
```

Comment:
I might also expect this to be mentioned on the export schema page, or other strategic locations. Is there a prompt of anything?
https://docs.yugabyte.com/preview/yugabyte-voyager/migrate/live-migrate/#assess-migration says it is optional, but it sort of isn't anymore. Probably needs rewording and is a good spot to mention it will run anyways.

Reply:
@makalaaneesh do we need to do it?

Reply:
We do a best effort  in terms of running assess-migration in export-schema. In other words, if it fails, export-schema is not aborted; it simply moves on. 
So, assess-migration is still optional. but for postgreSQL source, we also try and run assess-migration in export-schema itself, (if not already run). 

From the user's perspective, an assessment report is also generated when export-schema is run, so I agree, that it might be worth mentioning this. 
Perhaps, we can add a note in [export-schema section](https://docs.yugabyte.com/preview/yugabyte-voyager/migrate/migrate-steps/#export-schema) for all workflows: 
"For PostgreSQL source, if assess-migration is not already run, the schema is assessed and a migration assessment report is generated". 

cc: @sanyamsinghal @hbhanawat

Reply:
added the note in all workflows please check @makalaaneesh